runtime/rasdump/dmpsup.c

```diff
@@ -1206,8 +1206,8 @@ JVM_OnUnload(JavaVM *vm, void *reserved)
 }
 
 /**
- * On Linux the first call to get a backtrace can cause some initialisation work.
- * If this is called in a signal handler with other threads paused then one of
+ * On Linux and OSX the first call to get a backtrace can cause some initialisation
+ * work. If this is called in a signal handler with other threads paused then one of
```

Comment:
While you're changing this comment, can you fix the spelling of 'initialization' to be consistent with https://github.com/eclipse/openj9/pull/1734?

Reply:
fixed.